src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Integer.Unsigned.D.cs

```diff
@@ -43,7 +43,8 @@ private static bool TryFormatUInt64D(ulong value, byte precision, Span<byte> des
 
             if (leadingZeroCount > 0)
             {
-                FormattingHelpers.FillWithAsciiZeros(destination.Slice(0, leadingZeroCount));
+                // Fill with ASCII zeros
+                destination.Slice(0, leadingZeroCount).Fill((byte)'0');
```

Comment:
Note that Utf8Formatter is typically going to use this method for very small sizes. Is Array.Fill faster for these small sizes?